packages/epics/src/treasury/components/list-assets.tsx

```diff
@@ -0,0 +1,155 @@
+      name: 'Bitcoin',
+      symbol: 'BTC',
+      value: 5.25791,
+      usdEqual: 335887.76,
+      type: 'utility'
+    },
+    {
+      icon: 'https://s3-alpha-sig.figma.com/img/aa94/0f1c/549bd8537863aa03286c4d4bf31a73c7?Expires=1733702400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=TV0V4Qmi5qoN5vJWV9O3NKzPlMW4EHdAG1z~-rgex4o09Fb3ZloXcrlBxp64NQ4tJL6icDm~HxABqWXzBHDPjN-9lwYmdb1OtxgiByd0VpgX2-jPXUGG1bt5k9OYpW0VrjlLwMos~~pszLB~Q4XC-5mr5aWjrvPGGAUsLO9mvKNB3QwNo6DFlz5MAjQFgYx~34kumBluLgJ3KO2j0SdVQgzmHekpW2i3ZtFYF~K~P0Rz4zX-kJ6YTu003xPmZweIg4cU~D5bufYWecXCr2YhK7u1FKfG3N4EkhvGUZVADQo7j~U2fwJJ9jRvRF89gyBAz42-Apwpq-zUwMGHZq7NVA__',
+      name: 'Ethereum',
+      symbol: 'ETH',
+      value: 5.25791,
+      usdEqual: 335887.76,
+      type: 'liquid'
+    }
+  ]
+}
+
+export const ListAssets: React.FC<ListAssetsProps> = () => {
+  const formatValue = (num:number):string => {
+    const numStr = num.toString();
+    const [integerPart, decimalPart] = numStr.split('.');
+    const formattedIntegerPart = integerPart.replace(/\B(?=(\d{3})+(?!\d))/g, '.');
+    return decimalPart ? `${formattedIntegerPart},${decimalPart}` : formattedIntegerPart;
+  }
+  const [assets, setAssets] = useState(dummyData.assets);
+  const loadMoreAssets = () => {
+    const newAssets = [
+      {
+        icon: 'https://s3-alpha-sig.figma.com/img/245b/338d/4199c4b76377fa29775a7d395db0e05d?Expires=1733702400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=W0HmD12ptSTV7QwJnar5l0QXpYjN87EJIjR3Hqqh7tyEQzRKmqVynUAWjPNLrDte2kxbcXjxXxHCOq-hcg0Skzn3EVLtp~2MWHlU91cMZ49APOTsv4Q6s9il15KZfJftq45R4sgqvGzCT9ol-O8M795I3q1ironeDlCHKuK8nJfB8H8r4ECM8q3UU77GAvO~Us01N26MsnOdLYw3JI3PyWrQHZB95EjSEZSsJt7SQ7YX7D6NmCkEb~yIbYEVD3fEUF3wMmQWsUPGwgtELPtqiHYkKrkbhJl0ARYvLZ0fOJQOfiIY3vVQhB~zIpq8kEMkTNlVHNvI6rvOhf5WVmYYDA__',
+        name: 'New Bitcoin',
+        symbol: 'BTC',
+        value: 5.25791,
+        usdEqual: 335887.76,
+        type: 'utility'
+      },
+      {
+        icon: 'https://s3-alpha-sig.figma.com/img/245b/338d/4199c4b76377fa29775a7d395db0e05d?Expires=1733702400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=W0HmD12ptSTV7QwJnar5l0QXpYjN87EJIjR3Hqqh7tyEQzRKmqVynUAWjPNLrDte2kxbcXjxXxHCOq-hcg0Skzn3EVLtp~2MWHlU91cMZ49APOTsv4Q6s9il15KZfJftq45R4sgqvGzCT9ol-O8M795I3q1ironeDlCHKuK8nJfB8H8r4ECM8q3UU77GAvO~Us01N26MsnOdLYw3JI3PyWrQHZB95EjSEZSsJt7SQ7YX7D6NmCkEb~yIbYEVD3fEUF3wMmQWsUPGwgtELPtqiHYkKrkbhJl0ARYvLZ0fOJQOfiIY3vVQhB~zIpq8kEMkTNlVHNvI6rvOhf5WVmYYDA__',
+        name: 'New Bitcoin',
+        symbol: 'BTC',
+        value: 5.25791,
+        usdEqual: 335887.76,
+        type: 'utility'
+      },
+    ];
```

Comment:
to keep reviews simple, I'd recommend to move mock data out of the implementation and move it into `<component-name>.mock.ts` files.